### ISSUE TITLE
Score controllers' efficiency

### ISSUE THREAD

**Issue by erikquinn** [openscope/openscope#13]
A baseline could be calculated on how long it *should* take for a given aircraft to land or take off. Then compare it against how long a player actually took to make it happen, and we can provide some info about how they did. Would be quite helpful from the training perspective.

Originally reported in zlsa/atc#355.

**Comment by sixeyeco** [comment]
Agree. Along the same lines...there are many opportunities during play to bias either departures or takeoffs. It would be nice to have an indication of their expected landing/departure time. If this is not feasible, then perhaps a simpler system of lights could indicate their status.

![atc-improvement-suggestion-timer-lights-2](https://cloud.githubusercontent.com/assets/179496/26523886/29ca0310-42d7-11e7-87fc-7b684144a7c3.png)

A similar system could be adopted for ARRs; 

![atc-improvement-suggestion-timer-lights-arr1](https://cloud.githubusercontent.com/assets/179496/26524068/9589ed22-42dc-11e7-9b6f-089ea95aeff9.png)

A plane is not deemed to be late unless it is still "behind" at the time it locks on to ILS. "Slightly behind" tells controllers that they need to step it up and increase their speed. If a plane is still behind by the time locks on ILS only then is the LATE signal used.  

Rather than simply using a clock based on point of entry it would be ideal if the app could gauge the progress of arrivals. That is to say, that the app assesses the current route and speed and the lights indicate whether the plane is _expected_ to arrive on time given. I imagine that would certainly be tough, but it would let controllers know if their commands were sufficient to remedy the "behind" condition. 

I would **love** a reporting function that gives me data on my performance.
